ts/protocol/HSReplayDecoder.ts

```diff
@@ -177,7 +179,10 @@ class HSReplayDecoder extends Stream.Transform implements CardOracle {
 					id,
 					node.attributes.tags,
 					+node.attributes.playerID,
-					name
+					name,
+					undefined,
```

Comment:
We can probably remove cardId from the player constructor (and just super `undefined`). I can do that later though.